src/service/chat.js

```diff
@@ -189,7 +190,8 @@ const moveChatToAnon = async ({
       anon_user_info_emoji_code,
       anon_user_info_emoji_name,
       oldChannelId,
-      targetUserId
+      targetUserId,
+      source
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `source` parameter has been added to the payload sent to the API endpoint without any validation. It would be better to add a check for `source` before making the API call.
```diff
+    if (!source) {
+      throw new Error('Source must be provided');
+    }
     const response = await anonymousApi.post('/chat/move-to-anon', {
       anon_user_info_emoji_code,
       anon_user_info_emoji_name,
       oldChannelId,
       targetUserId,
       source
     });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->